packages/@uppy/dashboard/src/index.js

```diff
@@ -175,6 +172,13 @@ module.exports = class Dashboard extends Plugin {
     this.removeDragOverClassTimeout = null
   }
 
+  i18nInit () {
```

Comment:
We may want to override setOptions in these plugins to make this a bit easier to use
```js
setOptions (newOpts) {
  super.setOptions(newOpts)
  this.i18nInit()
}
```